docs/architecture/adr-template.md

```diff
@@ -1,5 +1,11 @@
 # ADR {ADR-NUMBER}: {TITLE}
 
+## Status
+
+> A decision may be "proposed" if it hasn't been agreed upon yet, or "accepted" once it is agreed upon. Once the ADR has been implemented mark the ADR as "implemented". If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement.
+
+{Deprecated|Proposed|Accepted|Declined|Implemented}
```

Comment:
this list doesn't contain "Deprecated" or "Superseded" either.

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1407/commits/be4369590deba49f6fab881aaa7988b22edfd2fe